test-files/cases/attribute/unifying/highway-2888/Expected.osm

```diff
@@ -5852,6 +5818,56 @@
         <tag k="hoot:status" v="3"/>
         <tag k="hoot:id" v="-156"/>
     </way>
+    <way visible="true" id="-155" timestamp="2019-01-24T18:58:28Z" version="1">
+        <nd ref="-96"/>
+        <nd ref="-151"/>
+        <nd ref="-798"/>
+        <nd ref="-797"/>
+        <nd ref="-921"/>
+        <nd ref="-920"/>
+        <nd ref="-919"/>
+        <nd ref="-918"/>
+        <nd ref="-917"/>
+        <nd ref="-916"/>
+        <nd ref="-915"/>
+        <nd ref="-914"/>
+        <nd ref="-913"/>
+        <nd ref="-967"/>
+        <nd ref="-966"/>
+        <nd ref="-965"/>
+        <nd ref="-964"/>
+        <nd ref="-963"/>
+        <nd ref="-962"/>
+        <nd ref="-961"/>
+        <nd ref="-960"/>
+        <nd ref="-959"/>
+        <nd ref="-958"/>
+        <nd ref="-957"/>
+        <nd ref="-973"/>
+        <nd ref="-972"/>
+        <nd ref="-971"/>
+        <nd ref="-970"/>
+        <nd ref="-969"/>
+        <nd ref="-968"/>
+        <nd ref="-947"/>
+        <nd ref="-946"/>
+        <tag k="source:imagery:datetime" v="2019-01-04 06:01:44"/>
+        <tag k="source:imagery:sensor" v="WV03_VNIR"/>
+        <tag k="bicycle" v="yes"/>
+        <tag k="security:classification" v="UNCLASSIFIED"/>
+        <tag k="length" v="12922.2"/>
+        <tag k="sidewalk" v="both"/>
+        <tag k="source:imagery" v="DigitalGlobe"/>
+        <tag k="source" v="DigitalGlobe"/>
+        <tag k="name" v="Boduthakurufaanu Magu"/>
+        <tag k="source:imagery:layerName" v="2019-01-04 - WV03_VNIR"/>
+        <tag k="foot" v="yes"/>
+        <tag k="source:datetime" v="2015-02-17T19:01:33.000Z;2019-01-24T18:58:28Z"/>
+        <tag k="source:imagery:id" v="5ce2e89d91bbea89ecf6dbf5915a50e2"/>
+        <tag k="highway" v="primary"/>
+        <tag k="hoot:status" v="3"/>
+        <tag k="hoot:id" v="-155"/>
+    </way>
```

Comment:
This is the very bottom way and it looks like it is more similar to the original geometry in the source data.  It looks like a good change to me.

Reply:
Right, the splits in the road close to the dual road section are closer to the reference than they were before.  Technically, neither of the divided roads should be picking up highway=primary or "Boduthakurufaanu Magu".  One of them already was doing this before this change anyway, and is on the list of things to fix ( 7) here: https://github.com/ngageoint/hootenanny/issues/2888#issuecomment-459465278).  I'd rather it be uniformly wrong on both sides of the dual roads for now anyway, instead of one side of the dual roads picking up tags and the other not picking up tags.